Update sector-pledging.md (#564)
filecoin-project/filecoin-docs: docs/mine/lotus/sector-pledging.md

## Upgrading pledged sectors

> The pledge period for the sector can be around 6-18 months(18 months by default). The _upgrading_ process allows however to recover pledged sectors for re-use much more quickly. The following command marks a sector for _upgrade_:

Reviewer: ```suggestion
The pledge period for the sector can be between 6 to 18 months. 18 months is the default. The _upgrading_ process allows however to recover pledged sectors for re-use much more quickly. The following command marks a sector for _upgrade_:
```
